subsys/bluetooth/host/smp.c

```diff
@@ -2275,6 +2290,14 @@ static u8_t get_auth(u8_t auth)
 		auth |= BT_SMP_AUTH_MITM;
 	}
 
+#if defined(CONFIG_BT_TESTING)
+	if (bondable) {
+		auth |= BT_SMP_AUTH_BONDING;
+	} else {
+		auth &= ~(BT_SMP_AUTH_BONDING);
```

Comment:
minor: the () seem unnecessary

Reply:
done. I just copied the code few lines above
		auth &= ~(BT_SMP_AUTH_MITM);
:)